deployer/src/handlers/mod.rs

```diff
@@ -609,6 +613,7 @@ async fn logs_websocket_handler(
 ) {
     let mut logs_request: tonic::Request<LogsRequest> = tonic::Request::new(LogsRequest {
         deployment_id: deployment_id.to_string(),
+        ..Default::default()
```

Comment:
We should consider allowing users to set a limit here too, but if it's tricky we can do it later. For example, in docker you can do `docker logs <id> --follow --tail 1000`. Although, tail is different from limit, perhaps we should consider supporting that rather than limit and page.